src/scenes/Authentication/Register/queries.graphql

```diff
@@ -0,0 +1,7 @@
+mutation Register($input: CreateUserInput!) {
```

Comment:
I'm thinking it would probably be better to name these files based on what's in them. i.e. `register.graphql`
It might make searching a bit easier...

Reply:
This will make for a way better developer experience. Renaming to `register.graphql`